wallet-adapter/packages/adapters/base-adapter/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "sui-base-wallet-adapter",
+  "version": "1.0.0",
+  "description": "Base wallet adapter for Sui",
+  "main": "lib/index.js",
+  "types": "lib/types/index.d.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

it might be better to add a build script before merging this, to make a bit easier for other to build it.

something like :
```suggestion
    "build": "tsc",
    "test": "echo \"Error: no test specified\" && exit 1"
```

and include typescript as a dev dependency